### PR TITLE
Bug/GH Committers Fail

### PR DESCRIPTION
**Pull request by nickmango**
- Resolved case for the pygithub query for accessing pr authors
- Updated pyGithub to 1.55
- Updated PyJWT to 2.3.0

Signed-off-by: Harold Wanyama <wanyaland+nickmango@gmail.com>

### CODE REVIEW

File: cla-backend/cla/models/github_models.py [hunk 1]
```diff
@@ -891,39 +891,45 @@ def get_pull_request_commit_authors(pull_request):
...
+                    cla.log.debug(f'PR: {pull_request.number}, GitHub commit.author.name and commit.author.login '
+                                f'author information NOT found for commit SHA {commit.sha}, '
+                                f'author id: {commit.author.id}, '
+                                f'name: {commit.author.name}, '
+                                f'login: {commit.author.login}, '
+                                f'email: {commit.author.email}')
+                    commit_authors.append((commit.sha, None))
+            except GithubException.IncompletableObject as ex:
+                cla.log.debug(f'Commit sha: {commit.sha} exception: {ex}')
+                if commit.commit.author is not None:
```
Comment (makkalot) [inline cla-backend/cla/models/github_models.py:920]: commit.commit can be None as well, add another check.

**Review by makkalot — COMMENTED** [review]
one small addition

File: cla-backend/cla/models/github_models.py (hunk 1 above)
Comment (nickmango) [reply cla-backend/cla/models/github_models.py:920]: Yes the None check is handled

Comment (makkalot) [reply cla-backend/cla/models/github_models.py:920]: `if commit.commit is not None and commit.commit.author is not None`

Comment (nickmango) [reply cla-backend/cla/models/github_models.py]: yea added a try catch block like the one earlier

**Pull request merged**